adoc/Harbor-Private-Registry-Scenarios.adoc

```diff
@@ -0,0 +1,226 @@
+== Supported Deployment Scenarios
+
+The following table lists the Helm chart deployment options officially supported by the SUSE Private Registry 2.1 release.
```

Comment:
I think this line should be right before the table, i.e. after the bash code block.